libs/pages/services/src/lib/feature/page-helm-create-feature/step-summary-feature/step-summary-feature.tsx

```diff
@@ -0,0 +1,187 @@
+} from '@qovery/shared/routes'
+import { Button, FunnelFlowBody, Heading, Icon, IconAwesomeEnum, Section } from '@qovery/shared/ui'
+import { useDocumentTitle } from '@qovery/shared/util-hooks'
+import { buildGitRepoUrl } from '@qovery/shared/util-js'
+import { useHelmCreateContext } from '../page-helm-create-feature'
+
+export function StepSummaryFeature() {
+  useDocumentTitle('Summary - Create Helm')
+
+  const { organizationId = '', projectId = '', environmentId = '' } = useParams()
+  const navigate = useNavigate()
+
+  const { generalForm, setCurrentStep } = useHelmCreateContext()
+  const generalData = generalForm.getValues()
+  setCurrentStep(3)
+
+  const pathCreate = `${SERVICES_URL(organizationId, projectId, environmentId)}${SERVICES_HELM_CREATION_URL}`
+
+  const { mutateAsync: createHelm, isLoading: isLoadingCreateHelm } = useCreateHelm()
+
+  const onSubmit = async (withDeploy: boolean) => {
+    const source = match(generalData.source_provider)
+      .with('GIT', () => {
+        const gitToken = getGitTokenValue(generalData.provider ?? '')
+
+        return {
+          git_repository: {
+            url: buildGitRepoUrl(gitToken?.type ?? generalData.provider ?? '', generalData.repository),
+            branch: generalData.branch,
+            root_path: generalData.root_path,
+          },
+        }
+      })
+      .with('HELM_REPOSITORY', () => ({
+        helm_repository: {
+          repository: generalData.repository,
+          chart_name: generalData.chart_name,
+          chart_version: generalData.chart_version,
+        },
+      }))
+      .exhaustive()
+
+    try {
+      await createHelm({
+        environmentId,
+        helmRequest: {
+          name: generalData.name,
+          description: generalData.description,
+          source,
+          allow_cluster_wide_resources: generalData.auto_preview!,
+          arguments: JSON.parse(generalData.arguments),
+          timeout_sec: generalData.timeout_sec,
+          auto_deploy: generalData.auto_deploy,
+          values_override: {},
+        },
+      })
+      navigate(SERVICES_URL(organizationId, projectId, environmentId) + SERVICES_GENERAL_URL)
+    } catch (error) {
+      console.log(error)
+    }
+  }
+
+  return (
+    <FunnelFlowBody>
+      <Section>
+        <Heading className="mb-2">Ready to create your Helm chart</Heading>
+        <p className="text-sm text-neutral-350 mb-10">
+          The setup is done, you can now create and deploy your Helm chart.
+        </p>
+
+        <div className="flex p-4 w-full border rounded border-neutral-250 bg-neutral-100 mb-10">
+          <Icon name={IconAwesomeEnum.CHECK} className="text-green-500 mr-2" />
+          <div className="flex-grow mr-2">
+            <div className="text-sm text-neutral-400 font-bold mb-5">Helm General Data</div>
+
+            <div className="text-neutral-400 text-ssm mb-2 font-medium">Helm chart general</div>
+
+            <ul className="text-neutral-350 text-sm list-none">
```

Comment:
Comment: This is a case for our `description-list` primitives. Unfortunately design isn't consistent so it can't be used as is. We need to ask design team to harmonize that.

Reply:
Yes, I mentioned it, 1 week ago, during the Design System and Alessandro told me to use the same with other services to avoid replace all Summary design for each